src/main/java/com/dev/cinema/controller/UserController.java

```diff
@@ -22,7 +25,7 @@ public UserController(UserService userService, UserMapper userMapper) {
     }
 
     @GetMapping("/by-email")
-    public UserResponseDto get(@RequestParam String email) {
+    public UserResponseDto get(@RequestParam @NotNull @Size(min = 4) @Valid String email) {
```

Comment:
As I know, `@NotNull` is redundant here, cause `@RequestParam` without `required` will throw exception if you send null as parameter. @Size is OK